### PR TITLE
docs: Add guide for snapshot_socket.

### PR DESCRIPTION
**Pull request by eiffel-fl**

### CODE REVIEW

**Review by burak-ok — COMMENTED** [review]
Could you also update the `artifacthub-pkg.yml` (home and source url)

File: gadgets/snapshot_socket/README.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+<TabItem value="ig" label="ig">
+```bash
+$ docker run --name test-snapshot-socket -d nginx
+...
+```
+</TabItem>
+</Tabs>
+
+Then, run the gadget and see how it shows the sleep socket:
```
Comment (blanquicet) [inline gadgets/snapshot_socket/README.mdx:48]:
```suggestion
Then, run the gadget and see how it shows the listening socket:
```

File: gadgets/snapshot_socket/README.mdx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    <TabItem value="ig" label="ig">
+        ```bash
+        $ sudo ig run ghcr.io/inspektor-gadget/gadget/snapshot_socket:latest [flags]
+        ```
+    </TabItem>
+</Tabs>
+
+## Guide
+
+Run a pod / container:
```
Comment (blanquicet) [inline gadgets/snapshot_socket/README.mdx:30]:
```suggestion
Run a nginx application:
```

**Review by blanquicet — APPROVED** [review]
Some nits but LGTM!

File: gadgets/snapshot_socket/README.mdx [hunk 3]
```diff
@@ -0,0 +1,87 @@
+---
+title: snapshot_socket
+---
```
Comment (mauriciovasquezbernal) [inline gadgets/snapshot_socket/README.mdx:3]: Please add a sidebar_position as done in https://github.com/inspektor-gadget/inspektor-gadget/commit/2e64540e15a111e92feb89d6140a0546de85b573 .

File: gadgets/snapshot_socket/README.mdx (hunk 1 above)
Comment (mauriciovasquezbernal) [inline gadgets/snapshot_socket/README.mdx:48]:
```suggestion
Then, run the gadget and see how it shows the socket used by nginx:
```

**Review by mauriciovasquezbernal — COMMENTED** [review]
Some minor fixes. Fell free to merge when you want.

**Pull request merged**